malasakit-django/pcari/admin.py

```diff
@@ -1,313 +1,46 @@
-            smart_str(u.age),
-            smart_str(u.barangay),
-            smart_str(u.gender),
-            smart_str(comment.id),
-            smart_str(comment.comment),
-            smart_str(comment.filipino_comment),
-            smart_str(comment.average_score),
-            smart_str(comment.number_rated),
-            smart_str(comment.date),
-            smart_str(comment.original_language),
-        ])
-    return response
-
-export_comment_csv.short_description = u"Export as CSV"
-
-def export_comment_xlsx(modeladmin, request, queryset):
-    response = HttpResponse(content_type='application/vnd.openxmlformats-officedocument.spreadsheetml.sheet')
-    response['Content-Disposition'] = 'attachment; filename=malasakit_comment_data.xlsx'
-    wb = openpyxl.Workbook()
-    ws = wb.get_active_sheet()
-    ws.title = "Malasakit Data"
-
-    row_num = 0
-
-    columns = [
-        (u"User", 15),
-        (u"Average_score", 70),
-        (u"Number_rated", 70),
-    ]
-
-    for col_num in xrange(len(columns)):
-        c = ws.cell(row=row_num + 1, column=col_num + 1)
-        c.value = columns[col_num][0]
-        c.style.font.bold = True
-        # set column width
-        ws.column_dimensions[get_column_letter(col_num+1)].width = columns[col_num][1]
-
-    for obj in queryset:
-        row_num += 1
-        row = [
-            obj.user,
-            obj.average_score,
-            obj.number_rated,
-        ]
-        for col_num in xrange(len(row)):
-            c = ws.cell(row=row_num + 1, column=col_num + 1)
-            c.value = row[col_num]
-            c.style.alignment.wrap_text = True
-
-    wb.save(response)
-    return response
-
-def dump_question_ratings_csv(modeladmin, request, queryset):
-    response = HttpResponse(content_type='text/csv')
-    response['Content-Disposition'] = 'attachment; filename=malasakit_question_rating_data.csv'
-    writer = csv.writer(response, csv.excel)
-    response.write(u'\ufeff'.encode('utf8')) # BOM (optional...Excel needs it to open UTF-8 file properly)
-    writer.writerow([
-        smart_str(u"User"),
-        smart_str(u"User Age"),
-        smart_str(u"User Barangay"),
-        smart_str(u"User Gender"),
-        smart_str(u"Question ID"),
-        smart_str(u"Score"),
-        smart_str(u"Date Created"),
-    ])
-    ratings = Rating.objects.all()
-    user_data = UserData.objects.all()
-    # users = User.objects.all()
-    for obj in ratings:
-        try:
-            u = user_data.filter(user=obj.user)[0]
-        except:
-            u = UserData()
 
-        writer.writerow([
-            smart_str(obj.user),
-            smart_str(u.age),
-            smart_str(u.barangay),
-            smart_str(u.gender),
-            smart_str(obj.qid),
-            smart_str(obj.score),
-            smart_str(obj.date),
-        ])
-    return response
+from .models import QualitativeQuestion, QuantitativeQuestion, QuantitativeQuestionRating, CommentRating, Comment, Respondent
```

Comment:
Probably cleaner to group related model imports on separate lines. (For instance, `QualitativeQuestion` and `QuantitativeQuestion`).